{"uuid": "45488e85-2519-4523-a279-973459539b6b", "children": ["e0fd5945-2405-4403-90cc-179eaff57525"], "befores": [{"name": "driver_double", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: Unable to receive message from renderer\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=126.0.6478.127)\nStacktrace:\n\tGetHandleVerifier [0x00007FF63AEEEEA2+31554]\n\t(No symbol) [0x00007FF63AE67ED9]\n\t(No symbol) [0x00007FF63AD2872A]\n\t(No symbol) [0x00007FF63AD10FDD]\n\t(No symbol) [0x00007FF63AD10CE0]\n\t(No symbol) [0x00007FF63AD0FC9B]\n\t(No symbol) [0x00007FF63AD3630D]\n\t(No symbol) [0x00007FF63ADBCCD5]\n\t(No symbol) [0x00007FF63AD9CDD3]\n\t(No symbol) [0x00007FF63AD6A33B]\n\t(No symbol) [0x00007FF63AD6AED1]\n\tGetHandleVerifier [0x00007FF63B1F8B1D+3217341]\n\tGetHandleVerifier [0x00007FF63B245AE3+3532675]\n\tGetHandleVerifier [0x00007FF63B23B0E0+3489152]\n\tGetHandleVerifier [0x00007FF63AF9E776+750614]\n\t(No symbol) [0x00007FF63AE7375F]\n\t(No symbol) [0x00007FF63AE6EB14]\n\t(No symbol) [0x00007FF63AE6ECA2]\n\t(No symbol) [0x00007FF63AE5E16F]\n\tBaseThreadInitThunk [0x00007FFCA48D7344+20]\n\tRtlUserThreadStart [0x00007FFCA5A7CC91+33]\n\n", "trace": "  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\conftest.py\", line 18, in driver_double\n    driver.get(Urls.HOME_PAGE)\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1720287516597, "stop": 1720287519666}], "start": 1720287516597, "stop": 1720287519706}